lib/session/call-session.js

```diff
@@ -1806,7 +1806,8 @@ Duration=${duration} `
       await writeAlerts({
         alert_type: AlertType.WEBHOOK_CONNECTION_FAILURE,
         account_sid: this.accountSid,
-        detail: `Session:reconnect error ${err}`
+        detail: `Session:reconnect error ${err}`,
+        url: this.application.call_hook.url,
```

Comment:
you can not simply add `url` to the alert if it is not part of the schema.  You should first add a field to the schema for url (in time-series)

Reply:
Oh sorry, I see you just want it logged, not added to the schema. Then this looks ok

Reply:
In other places where the same alert is created, the url is provisioned. Only in the "reconnection" case the `url` is missing

![Screenshot from 2024-07-31 15-06-33](https://github.com/user-attachments/assets/6db245c7-2d5c-4290-9be3-64ee161f8ca5)